src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-cycle */
+import React from 'react';
+import './TodoList.css';
+
+import { PreparedTodo } from '../../App';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todos: PreparedTodo[];
+};
+
+export const TodoList:React.FC<Props> = ({ todos }) => (
+  <ul className="todo__list">
+    {todos.map((todo) => (
+      <li className="todo__item" key={todo.id}>
+        {todo.user && (
+          <UserInfo user={todo.user} />
+        )}
```

Comment:
move it inside `TodoInfo`

Reply:
But my todo data is only in TodoList and I must import it in two files

Reply:
No it's not only in `TodoList`, you can easily move it

Reply:
Okey, I will keep that in mind in the future